static/app/views/releases/thresholdsList/index.tsx

```diff
@@ -157,12 +201,25 @@ function ReleaseThresholdList({}: Props) {
                       key={`${projId}-${envName}`}
                       thresholds={thresholdGroup}
                       refetch={refetch}
-                      columns={5}
                       orgSlug={organization.slug}
                       setError={tempError}
                     />
                   ));
                 })}
+              {selectedProjects.length === 1 &&
+                newThresholdGroup[0] &&
```

Comment:
Could this be empty at this point?

Reply:
Yes, if this is empty then we won't render anything

Reply:
I mean newThresholdGroup[0]. If it can be empty maybe `newThresholdGroup && newThresholdGroup[0]`

Reply:
Oh - so it'll always be an array (never falsey)
If it's an empty array, then we want this to be false, so no need to check for existence of `newThresholdGroup`